qcfractal/interface/tests/test_dataset.py

```diff
@@ -232,15 +231,15 @@ def test_nbody_rxn(nbody_ds):
 def test_database_history():
     ds = portal.collections.Dataset("history_test")
     history = [("energy", "p3", "m1", "b2", "o1"),
-               ("energy", "p1", "m1", None, "o1"),
-               ("energy", "p1", "m1", None, "o2"),
+               ("energy", "p1", "m1", "", "o1"),
+               ("energy", "p1", "m1", "", "o2"),
                ("energy", "p1", "m2", "b3", "o1"),
-               ("gradient", "p1", "m2", None, None)] # yapf: disable
+               ("gradient", "p1", "m2", "", "")] # yapf: disable
 
     for h in history:
         ds._add_history(driver=h[0], program=h[1], method=h[2], basis=h[3], keywords=h[4])
 
     assert ds.list_records().shape[0] == 5
     assert ds.list_records(program="P1").shape[0] == 4
-    assert ds.list_records(basis=None).shape[0] == 3
-    assert ds.list_records(keywords=None).shape[0] == 1
+    assert ds.list_records(basis="").shape[0] == 3
```

Comment:
Basis does need to be None for things like PM6/TorchANI/UFF/etc.

Reply:
`"None"` may be passed to search for fields which are `None`.